sensor_rotation:
  - 0.25774243915136663
  - -0.90060958645620659
  - -0.34995886593067532
  - 0.93873927848660577
  - 0.31916931889500888
  - -0.12999812653474357
  - 0.22877369185890031
  - -0.29501409908558018
  - 0.9277010721422998
force_bias:
  - 2.5300630639605135
  - 0.85629003419909688
  - -11.649664226958436
torque_bias:
  - -0.10004378985561634
  - 0.2955973880147626
  - 1.5028671110187867e-16
tool_mass: 0.29670225424725893
center_of_mass:
  - 4.458413548698872e-17
  - -7.2649540805644455e-17
  - 0.11683399999999974
gravity_force:
  - 1.443111733420025
  - 0.048499371732146968
  - -2.5272424896713197